superset/sql_lab.py

```diff
@@ -313,8 +313,6 @@ def execute_sql_statement(  # pylint: disable=too-many-arguments,too-many-statem
         if query.status == QueryStatus.STOPPED:
             raise SqlLabQueryStoppedException() from ex
 
-        logger.error("Query %d: %s", query.id, type(ex), exc_info=True)
-        logger.debug("Query %d: %s", query.id, ex)
```

Comment:
Can we keep the debug? Or does it also show up as exceptions?

Reply:
naw we end up eating it and converting to a `payload` i'll add it back